server/api/region.go

```diff
@@ -876,7 +875,7 @@ func (h *regionsHandler) ScatterRegions(w http.ResponseWriter, r *http.Request)
 			h.rd.JSON(w, http.StatusBadRequest, err.Error())
 			return
 		}
-		ops, failures, err = rc.GetRegionScatter().ScatterRegionsByRange(startKey, endKey, group, retryLimit)
+		opsCount, failures, err = rc.GetRegionScatter().ScatterRegionsByRange(startKey, endKey, group, retryLimit)
```

Comment:
how about return the success percentage directly? I do not see any where use the failures map?

Reply:
In fact, because of the retry mechanism， we need a map in ScatterRegions for count. we can reserve this map.